quantum.bst

```diff
@@ -1147,10 +1147,10 @@ FUNCTION {format.url}
   shorturl empty$
     { url empty$
         { "" }
-        { " url:~\href{" url * "}{" * "https://" #8 url chop.word * "}" * }
+	{ " url:~\url{" url * "}" * }
       if$
     }
-    { " url:~\href{" url * "}{" * "https://" #8 shorturl chop.word * "}" * }
+    { " url:~\url{" url * "}" * }
```

Comment:
This effectively deactivates the `shorturl` functionality, doesn't it? And `\url` does not allow us to display more human-readable versions of the content... Could `breakurl` be an option here? I'll look into this.

Reply:
Yes it does.